docs/web-configurator/web-configurator.mdx

```diff
@@ -4,7 +4,7 @@ title: GP2040-CE Web Configurator
 # -
 pagination_next: null
 pagination_prev: null
-description: "GP2040-CE's Web-Based Configuration Application: Just press S2 on boot and go to http://192.168.7.1"
+description: "GP2040-CE's Web-Based Configuration Application: Just hold S2 on boot or hold S2+B3+B4 while already plugged in. Then go to http://192.168.7.1 to access the Web Configurator."

```

Comment:
change to - highlighted inputs that are dynamic to selection.

Reply:
As this is the description that only appears for embed, the Label Selector cannot be used here.